build/lib/modernizr.min.js

```diff
@@ -1 +1,313 @@
\ No newline at end of file
+/*!
+ * Modernizr v2.8.3
```

Comment:
Unless GitHub is being clever to help me read this, is it (and other JS files in this PR) meant to no longer be minified?